filebeat/harvester/reader/multiline.go

```diff
@@ -150,9 +146,9 @@ func (mlr *Multiline) readNext() (Message, error) {
 
 				// return collected multiline event and
 				// empty buffer for new multiline event
-				message := mlr.finalize()
+				m := mlr.finalize()
```

Comment:
Agree. I used `m` because it is used in all other place. I changed all to `msg`.